xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -1033,6 +1033,18 @@ edit.defaultEditor.org.xwiki.rendering.block.XDOM#wysiwyg=$xwikiPropertiesDefaul
 #-# The default is :
 # notifications.async.poolSize = 2
 
+#-------------------------------------------------------------------------------------
+# Mentions
+#-------------------------------------------------------------------------------------
+
+#-# [Since 12.6RC1]
+#-# The detection and notification for the user mentions in created or edited content is done asynchronously 
+#-# by a pool of consumers.
+#-# This properties controls the size of this pool.
+#-#
+#-# The default is :
+# notifications.poolSize = 2
```

Comment:
Still not sure about this default size: it's something not critical so I'd allow the minimum of resources (i.e. 1 thread only). WDYT @tmortagne ?

Reply:
I don't think I would have even implemented a pool system :)